oss-internship-2020/pffft/README.md

```diff
@@ -0,0 +1,92 @@
+# Sandboxing PFFFT library
+
+Build System: CMake  
+OS: Linux
+
+### Check out the PFFFT library & CMake set up
+`git checkout -b master`
```

Comment:
This one is not necessary, as `master` will be the default that people check out